### PR TITLE
JMX reporter for other metrics

### PR DESCRIPTION
**Pull request by mproch**
Hi,
Please add changelog entry in docs/Changelog.md.
After that update description of this PR.
Thanks.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -242,6 +242,9 @@ lazy val commonSettings =
         "com.fasterxml.jackson.core" % "jackson-annotations" % jacksonV,
         "com.fasterxml.jackson.core" % "jackson-core" % jacksonV,
         "com.fasterxml.jackson.core" % "jackson-databind" % jacksonV,
+
+        "io.dropwizard.metrics5" % "metrics-core" % dropWizardV,
```
Comment (dswiecki) [inline build.sbt:246]: why do we use 5.0.0-rc3 exactly ? there's 5.0.0-rc11 already available ? https://search.maven.org/artifact/io.dropwizard.metrics5/metrics-core/5.0.0-rc11/bundle

File: build.sbt [hunk 2]
```diff
@@ -1414,6 +1418,10 @@ lazy val ui = (project in file("ui/server"))
 
         "com.dimafeng" %% "testcontainers-scala-scalatest" % testcontainersScalaV % "test",
         "com.dimafeng" %% "testcontainers-scala-postgresql" % testcontainersScalaV % "test",
+
+        "io.dropwizard.metrics5" % "metrics-core" % dropWizardV,
```
Comment (dswiecki) [inline build.sbt:1422]: don't we already have this in common ?

Comment (mproch) [reply build.sbt:1422]: nope. It's only in lite-runtime

File: build.sbt (hunk 1 above)
Comment (mproch) [reply build.sbt:246]: good point, I'll upgrade...

**Review by gadomsky — APPROVED** [review]
LGTM

**Pull request merged**